src/index.js

```diff
@@ -0,0 +1,155 @@
+  checkboxes.addEventListener('click', () => {
+    threeDots.classList.toggle('remove-icon-active');
+    trashIcon.classList.toggle('icon2');
+    listText.classList.toggle('listContent-disable');
+    list.classList.toggle('changeBg');
+    const getting = JSON.parse(localStorage.getItem('list'));
+    const empty = [];
+    const hammasi = document.querySelectorAll('.input-div');
+    for (let i = 0; i < getting.length; i += 1) {
+      if (hammasi[i].classList.contains('changeBg')) {
+        getting[i].completed = true;
+        count += 1;
+      } else {
+        getting[i].completed = false;
+      }
+      empty.push(getting[i]);
+      localStorage.setItem('list', JSON.stringify(empty));
+    }
+  });
+  const clearAll = document.querySelector('#clear');
+  clearAll.addEventListener('click', () => {
+    const getting = JSON.parse(localStorage.getItem('list'));
+    const variable = document.querySelectorAll('.changeBg');
+    for (let i = 0; i < variable.length; i += 1) {
+      form.removeChild(variable[i]);
+    }
+    const empty = [];
+    for (let i = 0; i < getting.length; i += 1) {
+      if (getting[i].completed === true) {
+        continue;
+      }
+      empty.push(getting[i]);
+    }
+    localStorage.setItem('list', JSON.stringify(empty));
+  });
+
+  // Remove from list event
+  trashIcon.addEventListener('click', () => {
+    form.removeChild(list);
+    const getFromLocalStorage = JSON.parse(localStorage.getItem('list'));
+    const result = getFromLocalStorage.filter((word) => word.description === listText.textContent);
+    const empty = [];
+    for (let i = 0; i < getFromLocalStorage.length; i += 1) {
+      if (result[0].description === getFromLocalStorage[i].description) {
+        continue;
+      }
+      empty.push(getFromLocalStorage[i]);
+    }
+    localStorage.setItem('list', JSON.stringify(empty));
+  });
+
+  threeDots.addEventListener('click', () => {
+    const editInput = document.createElement('input');
+    editInput.type = 'text';
+    editInput.className = 'listContent';
+    editInput.style.backgroundColor = '#fffed3';
+    list.style.backgroundColor = '#fffed3';
+    editInput.value = listText.textContent;
+    list.replaceChild(editInput, listText);
+    editInput.addEventListener('keypress', (e) => {
+      if (e.key === 'Enter' && editInput.value) {
+        const getting = JSON.parse(localStorage.getItem('list'));
+        const result = getting.filter((word) => word.description === listText.textContent);
+        const empty = [];
+        for (let i = 0; i < getting.length; i += 1) {
+          if (getting[i].index === result[0].index) {
+            getting[i].description = editInput.value;
+          }
+          empty.push(getting[i]);
+          localStorage.setItem('list', JSON.stringify(empty));
+        }
+        list.replaceChild(listText, editInput);
+        listText.textContent = editInput.value;
+        list.style.backgroundColor = '#fff';
+      }
+    });
+  });
+};
```

Comment:
- [ ] Your app should have a new JavaScript file for the new functionality (add, remove, and edit), kindly consider creating a new file and moving your code there so you meet all the requirement

Reply:
- Kindly see to this change it has not been implemented yet